CONTRIBUTING.md

```diff
@@ -283,6 +283,10 @@ update the [install](https://github.com/windmilleng/tilt.build/tree/master/docs/
 the [default dev version](internal/cli/build.go),
 and the [installer version](scripts/install.sh).
 
+The [upgrade docs](https://github.com/windmilleng/tilt.build/blob/master/docs/upgrade.md) 
```

Comment:
Fwiw this looks like an additional checklist item, which it isn't -- maybe put the "asap-ness" clause in the above paragraph.